sagemaker-inference-recommender/python-sdk-inference-recommender/inference-recommender-with-python-sdk.ipynb

```diff
@@ -0,0 +1,652 @@
+{
```

Comment:
<ol><li>Is this model downloaded from public S3? If it is, could we just retrieve it instead of adding model in the folder? like what we did in this <a href="https://github.com/aws/amazon-sagemaker-examples/blob/main/sagemaker-inference-recommender/sklearn-inference-recommender/sklearn-inference-recommender.ipynb" target="_blank">notebook</a>.</li><li>nit: usually we will put the inference.py under a <code>code</code> folder, check <a href="https://github.com/aws/amazon-sagemaker-examples/tree/main/sagemaker-inference-recommender/sklearn-inference-recommender/code" target="_blank">here</a>. We could do the same for sample payload too, likewise in <a href="https://github.com/aws/amazon-sagemaker-examples/tree/main/sagemaker-inference-recommender/huggingface-inference-recommender/sample-payload" target="_blank">huggingface example</a>.</li></ol>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3767/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>